constants.py

```diff
@@ -329,6 +334,10 @@
 
 kRobotPoseArrayKeys = OptionalValueKeys("RobotPoseArray")
 
+kDriveVelocityKeys = "robotVelocity"
+kRobotUpdateRate = 1 / 50
```

Comment:
Rename to `kRobotUpdatePeriod` (a period is measured in seconds, a rate/frequency is measured in Hz)